sydent/http/servlets/__init__.py

```diff
@@ -92,9 +103,24 @@ def get_args(request, required_args):
 
     return args
 
+
 def jsonwrap(f):
     @functools.wraps(f)
     def inner(self, request, *args, **kwargs):
+        """
+        Runs a web handler function with the given request and parameters, then
+        converts its result in JSON and returns it. If an error happens, also sets
+        the HTTP response code.
+
+        :param self: The current object.
+        :param request: The request to process.
+        :type request: twisted.web.server.Request
+        :param args: The arguments to pass to the function.
+        :param kwargs: The keyword arguments to pass to the function.
```

Comment:
for clarity: this stuff is great and I think it's always worth writing down once you've taken the time to figure it out, but I wouldn't see it as a requirement for porting to python3. it would be entirely reasonable to just add the `:return:` and `:rtype` lines.